ledger/apply/application.go

```diff
@@ -118,7 +118,7 @@ func createApplication(ac *transactions.ApplicationCallTxnFields, balances Balan
 	// Update the cached TotalExtraAppPages for this account, used
 	// when computing MinBalance
 	totalExtraPages := record.TotalExtraAppPages
-	totalExtraPages += ac.ExtraProgramPages
+	totalExtraPages = basics.AddSaturate32(totalExtraPages, ac.ExtraProgramPages)
```

Comment:
shouldn't we err in case of an overflow ? ( i.e. I know that this one is extremely theoretical )